packages/cli/cli/versions.yml

```diff
@@ -1,3 +1,11 @@
+- changelogEntry:
+  - summary: |
+        Docs configuration now lists all the top level roles, and OAuth Client Credential
+        configuration is now more expressive.
```

Comment:
@RohinBhargava can you add a yaml example of how to set the field? 

And do you mind adding a section at the bottom of https://buildwithfern.com/learn/api-definition/fern/authentication#oauth-client-credentials to explain how to configure the header

Reply:
will do!